test/show

```diff
@@ -64,43 +65,43 @@ case "${MY_scope}" in
     case "${MY_item}" in
       'nodes')
         if [ "${MY_format}" == 'json' ]; then
-          kubectl -o json get nodes "${@}" \
+          ${KUBECTL} -o json get nodes "${@}" \
```

Comment:
Work with an out-of-PATH version using the ad-hoc environment variable KUBECTL like described [here](https://github.com/exoscale/exoscale-cloud-controller-manager/tree/master/test#kubectl)